packages/hmr-plugin/src/symbols.ts

```diff
@@ -0,0 +1,21 @@
+import { StateStream } from '@ngxs/store';
+
+export interface NgxsStoreSnapshot {
+  [key: string]: any;
+}
```

Comment:
Do we need an interface for the snapshot?

Reply:
We need to understand that we are returning a state